<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reg</class>
 <widget class="QDialog" name="reg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>461</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="email"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="loginL">
      <property name="text">
       <string>Логин:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="login"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="password"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="emailL">
      <property name="text">
       <string>Еmail:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="passwordL">
      <property name="minimumSize">
       <size>
        <width>56</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>Пароль:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="passwordL_2">
      <property name="text">
       <string>Повторите пароль:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="password_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>331</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="back">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="reg_2">
      <property name="text">
       <string>Зарегестрироваться</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
